src/components/MultiGestureCanvas/types.ts

```diff
@@ -39,9 +41,10 @@ type MultiGestureCanvasVariables = {
     panTranslateY: SharedValue<number>;
     pinchTranslateX: SharedValue<number>;
     pinchTranslateY: SharedValue<number>;
-    stopAnimation: WorkletFunction<[], void>;
-    reset: WorkletFunction<[boolean], void>;
-    onTap: OnTapCallback;
+    stopAnimation: () => void;
+    reset: (animated: boolean) => void;
+    onTap: OnTapCallback | undefined;
+    onScaleChanged: OnScaleChangedCallback | undefined;
```

Comment:
```suggestion
    onTap?: OnTapCallback;
    onScaleChanged?: OnScaleChangedCallback;
```

Reply:
Imo, this looks like passing some of the variables is optional, but it actually isn't. Even if it's undefined, we still wan't to enforce passing it to the hooks (like `useTapGesture`.

Reply:
If we remove the `?`, it will throw an error here:

<img width="1534" alt="Screenshot 2024-01-17 at 10 53 13" src="https://github.com/Expensify/App/assets/20173411/c0bee680-7310-4b10-a025-08fad388ee1b">